core/src/validator.rs

```diff
@@ -249,6 +250,48 @@ impl TransactionStructure {
     }
 }
 
+#[derive(
+    Clone, Debug, EnumVariantNames, IntoStaticStr, Display, Serialize, Deserialize, PartialEq, Eq,
+)]
+#[strum(serialize_all = "kebab-case")]
+#[serde(rename_all = "kebab-case")]
+pub enum SchedulerPacing {
+    Disabled,
+    FillTimeMillis(NonZeroU64),
+}
+
+impl Default for SchedulerPacing {
```

Comment:
i'd avoid (and the derive on config container above)

Reply:
> Though then banking stage should take the whole config as input instead of SchedulerPacing

you'll be happy when you have to add the second arg to config and it's a one-liner instead of a replumb